mods/xpanes/license.txt

```diff
@@ -34,6 +34,7 @@ Attribution-ShareAlike 3.0 Unported (CC BY-SA 3.0)
 Copyright (C) 2014-2016 xyz
 Copyright (C) 2013-2016 Gambit
 Copyright (C) 2016 paramat
+copyright (C) 2016-2019 TumeniNodes
```

Comment:
Copyright

Reply:
Also, why from 2016? Aren't the textures made recently?

Reply:
yes, corrected.